configs/components/sysroot.rb

```diff
@@ -8,7 +8,7 @@
       pkg.md5sum "d08a22cd4f431d4e1fc0d1869f60d9da"
     when "el-7-aarch64"
       pkg.version "2017.06.13"
```

Comment:
Just a question, and shouldn't block a merge - would we want to consider incrementing `pkg.version` here?